### ISSUE TITLE
Reproduce results - block style

### ISSUE THREAD

**Issue by guy-ealey-morag** [OfirKedem/Pointer-Value-Retrieval#8]
- **Reproduce Figure 6**
- [ ] VGG IID
- [ ] ResNet IID

- **Reproduce Figure 7**
- [ ] VGG DShift
- [ ] ResNet DShift
- [ ] VGG Holdout
- [ ] ResNet Holdout

- **Reproduce Figure 8**
- [ ] IDD Shift logits
- [ ] Holdout Shift logits
